seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/StringTemplate.java

```diff
@@ -37,7 +37,7 @@ public static String substitute(String str, String timeFormat) {
         final SimpleDateFormat sdf = new SimpleDateFormat(timeFormat);
         final String formattedDate = sdf.format(new Date());
 
-        final Map<String, String> valuesMap = new HashMap(5);
+        final Map<String, String> valuesMap = new HashMap(3);
```

Comment:
hi @asdf2014, it seems not necessary to modify it.

Reply:
Yes, there will be resize

Reply:
Hi @leo65535 @CalvinKirs, thanks for your comment, I agree with you, but if we use 5, it will resize to 8, which is unsuitable. If we use 3, it will resize to 4, which is quit enough here. So, I believe this improvement is necessary, right

Reply:
Sorry,ignore me, i remember wrong, the resize is only when the fourth element is added

Reply:
@CalvinKirs No worries 😄